### PR TITLE
add slides section to vancouver agenda

### PR DESCRIPTION
**Pull request by ErickWendel**
#132

### CODE REVIEW

File: 2018-10-Vancouver/agenda.md [hunk 1]
```diff
@@ -64,31 +64,31 @@ _We are expecting 70-100 people to participate_.
...
+| 2:45 - 3:45   | Breakout 1 (Room 118)   | Governance                                       |                               | | |
+| 2:45 - 3:45   | Breakout 2 (Room 119)        | Benchmarking | @davisjam | https://github.com/nodejs/summit/issues/120 | |
+| 3:45          |              | __Break__                                                           |                               | | |
+| 4:00          | Main (Room 121)        | __Lightning talks__                                                  |                               | | |
+|               |              | Deep dive on Timers                                                 | @apapirovski                  | | |
+|               |              | Diagnostics at Netflix                                              | @hekike                       | | |
+|               |              | Academic perspectives on Node.js | @davisjam | |
+|               |              | Navigating Node.js C++ code | @addaleax | | |
```
Comment (addaleax) [inline 2018-10-Vancouver/agenda.md]: If you want to add it to the PR, https://addaleax.net/node-cpp-vancouver/

**Comment by davisjam** [comment]
I can open my own PR once this lands, or:
- [slides](http://people.cs.vt.edu/~davisjam/downloads/misc/Davis-NodeJSInteractive-AcademicPerspectivesOnNode.pptx) for "academic perspectives" talk
- [slides](http://people.cs.vt.edu/~davisjam/downloads/misc/Davis-NodeJSInteractive-BenchmarkingAndPerformanceBreakout.pptx) for benchmarking session

**Comment by ErickWendel** [comment]
> I can open my own PR once this lands, or:
> 
> * [slides](http://people.cs.vt.edu/%7Edavisjam/downloads/misc/Davis-NodeJSInteractive-AcademicPerspectivesOnNode.pptx) for "academic perspectives" talk
> * [slides](http://people.cs.vt.edu/%7Edavisjam/downloads/misc/Davis-NodeJSInteractive-BenchmarkingAndPerformanceBreakout.pptx) for benchmarking session

Now is on commit :D

File: 2018-10-Vancouver/agenda.md [hunk 2]
```diff
@@ -64,31 +64,31 @@ _We are expecting 70-100 people to participate_.
...
+| 9:00          | Main (Room 121)        | Intro, Code of Conduct                                              | @mcollina @dshaw @chowdhurian |                                            | |
+| 9:15 - 10:45  | Main (Room 121)        | Modules                                                             | @MylesBorins                  | https://github.com/nodejs/modules/issues/177 | |
+| 9:15 - 10:45  | Breakout 1 (Room 118)   | Website Redesign                                                    | @amiller-gh                   | https://github.com/nodejs/website-redesign/issues/88 | |
+| 9:15 - 10:45  | Breakout 2 (Room 119)   | Security WG                                      |        @mhdawson         | https://github.com/nodejs/security-wg/issues/358 | |
+| 10:45         |              | __Coffee Break__                                                    |                               |  | |
+| 11:00 - 12:30 | Main (Room 121)        | Promises                                      | @BridgeAR                              | | [Code examples](https://gist.github.com/mmarchini/1f7476e0ee21a1f17832971289c3f4b5)|
```
Comment (mmarchini) [inline 2018-10-Vancouver/agenda.md]: This gist was presented in the Diagnostics session. Can you move it there?

File: 2018-10-Vancouver/agenda.md [hunk 3]
```diff
@@ -26,23 +26,23 @@ _We are expecting 300 people to participate_.
...
+| 3:30          | Main         | Mentorship                                    | @Bamieh                                | | |
+| 3:30          | Main         | LTS + release                                          |    @MylesBorins          | https://github.com/nodejs/Release/issues/358  | |
+| 3:30          | Main         | Tooling                                          | @boneskull                                       | https://github.com/nodejs/tooling/issues/4 | |
+| 3:30          | Main         | Embedding                                     | @codebytere                       | https://github.com/nodejs/summit/issues/125   | [The Embedder's POV](https://github.com/codebytere/gives-talks/blob/master/node-collab-summit-2018/the_embedders_pov.pdf)|
+| 3:30          | Breakout 1 (Room 118)   | Diagnostics                                   | @kjin , @yunong                   | https://github.com/nodejs/diagnostics/issues/229 | [Diagnostics](https://docs.google.com/a/google.com/presentation/d/e/2PACX-1vS19WYtuxxx6I7_Y7Qy7c4wLARbKEcveYsY-8XgBes0YTVw70j3x22KBltaUGGTNa73TIET6Vvx7Cw7/pub?start=false&loop=false&delayms=3000)|
```
Comment (mmarchini) [inline 2018-10-Vancouver/agenda.md]: @kjin I'm getting "You need permission to access this published document" for the Diagnostics Session slides.

File: 2018-10-Vancouver/agenda.md (hunk 2 above)
Comment (ErickWendel) [reply 2018-10-Vancouver/agenda.md]: Yeah! I’ll do today :D

File: 2018-10-Vancouver/agenda.md [hunk 4]
```diff
@@ -64,31 +64,31 @@ _We are expecting 70-100 people to participate_.
...
+|---------------|--------------|---------------------------------------------------------------------|-------------------------------|--------------------------------------------|-------------------------------------------|
+| 8:00          |              | Breakfast and registration                                          |                               |                                            | |
+| 9:00          | Main (Room 121)        | Intro, Code of Conduct                                              | @mcollina @dshaw @chowdhurian |                                            | |
+| 9:15 - 10:45  | Main (Room 121)        | Modules                                                             | @MylesBorins                  | https://github.com/nodejs/modules/issues/177 | |
+| 9:15 - 10:45  | Breakout 1 (Room 118)   | Website Redesign                                                    | @amiller-gh                   | https://github.com/nodejs/website-redesign/issues/88 | |
```
Comment (sagirk) [inline 2018-10-Vancouver/agenda.md]: Slides for the website redesign breakout, if you want to add them.

```suggestion
| 9:15 - 10:45  | Breakout 1 (Room 118)   | Website Redesign                                                    | @amiller-gh                   | https://github.com/nodejs/website-redesign/issues/88 | [Website Redesign: A History](https://docs.google.com/presentation/d/1kHsqeQxg1lxWUvz4pVENsK9wY4FFFzeUORD-UwYfrNo) |
```

File: 2018-10-Vancouver/agenda.md (hunk 3 above)
Comment (kjin) [inline 2018-10-Vancouver/agenda.md]:
```suggestion
| 3:30          | Breakout 1 (Room 118)   | Diagnostics                                   | @kjin , @yunong                   | https://github.com/nodejs/diagnostics/issues/229 | [Diagnostics](https://docs.google.com/presentation/d/1lFkqkTMbSad7Km-CRiCGcd4Wcwnc18wOkWsbuMKB1Jo)|
```
Sorry for the late reply. This link should work

File: 2018-10-Vancouver/agenda.md [hunk 5]
```diff
@@ -26,23 +26,23 @@ _We are expecting 300 people to participate_.
...
+| 3:15          | Main         | Session presentation, set up the Main room    | @mcollina @dshaw @chowdhurian          | | |
+| 3:30          | Main         | Moderation                                    | @hackygolucky                          | | |
+| 3:30          | Main         | Mentorship                                    | @Bamieh                                | | |
+| 3:30          | Main         | LTS + release                                          |    @MylesBorins          | https://github.com/nodejs/Release/issues/358  | |
+| 3:30          | Main         | Tooling                                          | @boneskull                                       | https://github.com/nodejs/tooling/issues/4 | |
+| 3:30          | Main         | Embedding                                     | @codebytere                       | https://github.com/nodejs/summit/issues/125   | |
+| 3:30          | Breakout 1 (Room 118)   | Diagnostics                                   | @kjin , @yunong                   | https://github.com/nodejs/diagnostics/issues/229 | [Diagnostics](https://docs.google.com/presentation/d/1lFkqkTMbSad7Km-CRiCGcd4Wcwnc18wOkWsbuMKB1J)|
```
Comment (joyeecheung) [inline 2018-10-Vancouver/agenda.md]: This link is missing an `o` at the end

```suggestion
| 3:30          | Breakout 1 (Room 118)   | Diagnostics                                   | @kjin , @yunong                   | https://github.com/nodejs/diagnostics/issues/229 | [Diagnostics](https://docs.google.com/presentation/d/1lFkqkTMbSad7Km-CRiCGcd4Wcwnc18wOkWsbuMKB1Jo)|
```

Comment (ErickWendel) [reply 2018-10-Vancouver/agenda.md]: added :D

**Pull request merged**